ios/PrideLondonApp.xcodeproj/project.pbxproj

```diff
@@ -347,6 +347,13 @@
 			remoteGlobalIDString = 3D3CD9181DE5FBD800167DC4;
 			remoteInfo = "jschelpers-tvOS";
 		};
+		4F3F4DC620C68433004B6EF1 /* PBXContainerItemProxy */ = {
```

Comment:
This file has likely changed because of RN link but because it's so hard to actually know we generally just let these changes go in 🤷‍♀️. Just an FYI